server/services/star.js

```diff
@@ -82,12 +82,34 @@ module.exports = class StarService extends EventEmitter {
         homeStar.warpGate = false;
         homeStar.specialistId = null;
 
-        if (!homeStar.isAsteroidField) {
-            homeStar.naturalResources = game.constants.star.resources.maxNaturalResources; // Home stars should always get max resources.
+        if (game.settings.specialGalaxy.splitResources && game.settings.specialGalaxy.splitResources === 'enabled') {
```

Comment:
The logic in this if statement would be better placed in a `starResourceService`.